worker/handler/handler.go

```diff
@@ -277,6 +338,21 @@ func (h *Handler) RunStart() (ch *sb.Channel, err error) {
 
 	// create sandbox if needed
 	if h.sandbox == nil {
+		// if estimated mem usage exceeds the hard limit, we would block 
```

Comment:
This model feels wrong to me (everybody Get invocation is effectively doing its own polling, and which get to proceed when resources become free is arbitrary).

When we're waiting to proceed, shouldn't we Get be added to a queue, and then a scheduler (implemented by a separate go routine) decides which Get calls to unblock, and when?

Reply:
That makes sense, updated in the new commit.